qiskit/primitives/utils.py

```diff
@@ -15,16 +15,18 @@
 from __future__ import annotations
 
 from collections.abc import Iterable
+from typing import Sequence
```

Comment:
Should we use `collections.abc` instead of `typing` here?

Reply:
Oh... What’s wrong with me